components/atoms/RadioCheck/radio-check.tsx

```diff
@@ -14,6 +14,14 @@ const RadioCheck = ({ className, id, children, value, checked = false, onClick }
   return (
     <div
       onClick={onClick}
+      onKeyDown={(e) => {
```

Comment:
This should be an actual button, and we can remove the `tabIndex` and `role="button"`